diff --git a/node_modules/@neos-project/build-essentials/src/postcss.config.js b/node_modules/@neos-project/build-essentials/src/postcss.config.js
index 7d74e8f..17b22e5 100644
--- a/node_modules/@neos-project/build-essentials/src/postcss.config.js
+++ b/node_modules/@neos-project/build-essentials/src/postcss.config.js
@@ -4,9 +4,9 @@ const styleVars = styles.generateCssVarsObject(styles.config);
 module.exports = {
     plugins: [
         require('autoprefixer'),
-        require('postcss-css-variables')({
-            variables: Object.assign(styleVars)
-        }),
+        // require('postcss-css-variables')({
+        //     variables: Object.assign(styleVars)
+        // }),
         require('postcss-import')(),
         require('postcss-nested')(),
         require('postcss-hexrgba')()
